source/reporters/ut_realtime_reporter.tpb

```diff
@@ -57,25 +57,27 @@ create or replace type body ut_realtime_reporter is
       self.print_node('name', a_suite.name);
       self.print_node('description', a_suite.description);
       <<suite_elements>>
+      self.print_start_node('items');
       for i in 1 .. a_suite.items.count loop
         if a_suite.items(i) is of(ut_test) then
           print_test_elements(treat(a_suite.items(i) as ut_test));
         elsif a_suite.items(i) is of(ut_logical_suite) then
           print_suite_elements(treat(a_suite.items(i) as ut_logical_suite));
         end if;
       end loop suite_elements;
+      self.print_end_node('items');
       self.print_end_node('suite');
     end print_suite_elements;
   begin
     xml_header := ut_utils.get_xml_header(a_run.client_character_set);
     self.print_xml_fragment(xml_header);
     self.print_start_node('event', 'type', 'pre-run');
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Define a constant instead of duplicating this literal 12 times. ([plsql:S1192](https://sonarcloud.io/organizations/utplsql/rules#rule_key=plsql%3AS1192))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=utPLSQL&pullRequest=819&issues=AWhDYNbDXONnvcWDUk4h&open=AWhDYNbDXONnvcWDUk4h)

[ISSUE_KEY]: AWhDYNbDXONnvcWDUk4h